### ISSUE TITLE
Keybinding to copy a line's results

### ISSUE THREAD

**Issue by jwiegley** [sulami/literate-calc-mode.el#46]
When I don't have results being inserted into the buffer, is there any way to quickly copy the results shown for the current line onto the kill ring? I find myself often wanting to paste this value into calc to do further work on it, so even a `literate-calc-result-to-calc` would be pretty awesome...

Loving this mode! It has replaced my use of Soulver 3 now. :)

**Comment by sulami** [comment]
Good idea. I've added pushing to the kill ring, which is pretty simple. To push into `calc`, I'd have to dig into that a bit to figure out how to properly insert a value.
